compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -406,6 +456,106 @@ void OMR::CodeGenerator::lowerTrees()
       }
 
    self()->postLowerTrees();
+
```

Comment:
Similar to comment above, please move this to `postLowerTrees()`.

Reply:
Addressed above.

Reply:
Done